lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorRequestBuilder.cs

```diff
@@ -93,7 +93,11 @@ public EvaluatorRequestBuilder SetRackName(string rackName)
             return this;
         }
 
-        // TODO[REEF-718]: Document.
+        /// <summary>
```

Comment:
Here_evaluatorBatchId is per builder therefore per request. How do you ensure all requests in the same batch share the same bath id?